etl/steps/data/garden/un/2024-10-21/census_dates.excluded_countries.json

```diff
@@ -0,0 +1,2 @@
+[
```

Comment:
you don't need this file if you aren't excluding any countries so you can remove it here and the link to it in the garden step

Reply:
Ah good point, thanks!